public/css/style.css

```diff
@@ -0,0 +1,62 @@
+
+.comment-list .row {
+    margin-bottom: 0;
+}
+.comment-list .panel .panel-heading {
+    padding: 4px 15px;
+    position: absolute;
+    border:none;
+    border-top-right-radius:0;
+    top: 1px;
+}
+.comment-list .panel .panel-heading.right {
+    border-right-width: 0;
+    border-top-left-radius:0;
+    right: 16px;
+}
+.comment-list .panel .panel-heading .panel-body {
+    padding-top: 6px;
+}
+.comment-list figcaption {
+    word-wrap: break-word;
+}
+@media (min-width: 768px) {
+    .comment-list .arrow:after, .comment-list .arrow:before {
+        content: "";
+        position: absolute;
+        width: 0;
+        height: 0;
+        border-style: solid;
+        border-color: transparent;
+    }
+    .comment-list .panel.arrow.left:after, .comment-list .panel.arrow.left:before {
+        border-left: 0;
+    }
+    .comment-list .panel.arrow.left:before {
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=5982252)